x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -520,9 +522,21 @@ export enum MetadataQueryStrategyVersions {
   VERSION_2 = 'v2',
 }
 
+export type PolicyInfo = Immutable<{
+  revision: number;
+  id: string;
+}>;
+
 export type HostInfo = Immutable<{
   metadata: HostMetadata;
   host_status: HostStatus;
+  policy_info?: {
+    agent: {
+      configured: PolicyInfo; // as set in kibana
```

Comment:
suggestion that you add these comments as block comments above the property so that we get it on content assist?

Reply:
TIL that's how that works. thanks